lib/active_merchant/billing/gateways/moneris.rb

```diff
@@ -438,6 +438,19 @@ def wallet_indicator(token_source)
         }[token_source]
       end
 
+      def format_order_id(wallet_indicator_code, order_id = nil)
+        # Truncate (max 100 characters) order id for
+        # google pay and apple pay (specific wallets / token sources)
+        wallets = %w(APP GPP)
```

Comment:
Can we take this out of the method and make it a class variable?

Reply:
Yes